TypeScript bindings broken: error TS2094: The property 'TilemapRenderer' does not exist on value of type 'Phaser'.

`tsc -t ES5 phaser.d.ts`

yields:

`phaser.d.ts(2024,26): error TS2094: The property 'TilemapRenderer' does not exist on value of type 'Phaser'.`

the version of tsc I am using is 0.9.5.0.

It looks like commit 10252543f2035cc1e11be444f66f2fe4922a16b1 removed the definition of TilemapRenderer.

best regards
-- Benjamin

--- a/build/phaser.d.ts
+++ b/build/phaser.d.ts
@@ -2375,7 +2375,6 @@ declare module Phaser {
         images: any[];
         key: string;
         layers: Phaser.TilemapLayer[];
-//review
         objects: any[];
         orientation: string;
         properties: Object;
@@ -2395,7 +2394,6 @@ declare module Phaser {
         visible: boolean;
         position: Phaser.Point;
         type: number;
-        renderer: Phaser.TilemapRenderer;
         mapFormat: string;
         //methods
         addTilesetImage(tileset: string, key?: string): void;
@@ -2501,16 +2499,6 @@ declare module Phaser {
         static parseCSV(data: string): Phaser.Tilemap;
         static parseJSON(json: Object): Phaser.Tilemap;
         static tileset(game: Phaser.Game, key: string, tileWidth: number, tileHeight: number, tileMargin?: number, tileSpacing?: number, rows?: number, colums?: number, total?: number): Phaser.Tileset;
-    }
-
-    //may be unused
-    class TilemapRenderer {
-        //constructor
-        constructor(game: Phaser.Game);
-        //members
-        game: Phaser.Game;
-        //methods
-        render(tilemap: Tilemap): void;
     }
 
     class Tileset {

